src/azure-cli/azure/cli/command_modules/acr/connected_registry.py

```diff
@@ -76,7 +76,7 @@ def acr_connected_registry_create(cmd,  # pylint: disable=too-many-locals, too-m
                        "Enabling the data endpoint might affect your firewall rules.\nTo enable data endpoint run:" +
                        "\n\taz acr update -n {} --data-endpoint-enabled true".format(registry_name))
 
-    ErrorResponseException = cmd.get_models('ErrorResponseException')
+    from azure.core.exceptions import HttpResponseError as ErrorResponseException
```

Comment:
Is the type same as before?

Reply:
Basically, yes. Track2 SDK has removed all these Exceptions in models.